include/lib/stdlib/string.h

```diff
@@ -55,6 +55,7 @@ void	*memcpy(void * __restrict, const void * __restrict, size_t);
 void	*memcpy16(void * __restrict, const void * __restrict, size_t);
 void	*memmove(void *, const void *, size_t);
 void	*memset(void *, int, size_t);
+void	 zeromem16(void *, unsigned int);
```

Comment:
The zeromem16 function is not one that we want to export - we currently using it internally only due to its focused alignment requirements.
Options are

- Use the new, more general purpose zeromem functions in utils.h
- Or if you wish to keep using the zeromem16, keep the extern declaration where it currently is